src/main/java/com/microsoft/azure/helium/HeliumJavaApplication.java

```diff
@@ -0,0 +1,20 @@
+package com.microsoft.azure.helium;
```

Comment:
Is this the correct namespace? not really part of microsoft.azure

Reply:
Following convention from the old repo. However, you have a great point as this is not official azure. Should we change it to com.microsoft.cse.helium? or simply com.microsoft.helium?

Reply:
Barring any existing examples from which we can borrow namespace, I'd go with com.microsoft.cse.helium as it gives us a namespace to use for other things (e.g., com.microsoft.cse.[Java app]).

Reply:
Agreed. I am changing to com.microsoft.cse